gapic/src/main/com/google/gapid/LoadingScreen.java

```diff
@@ -91,8 +90,7 @@ public LoadingScreen(Composite parent, Theme theme) {
     statusLabel = createLabel(container, "Starting up...");
     statusLabel.setLayoutData(new GridData(SWT.CENTER, SWT.TOP, true, false));
 
-    optionsContainer = createComposite(
-        container, withSpacing(filling(new RowLayout(SWT.VERTICAL), true, true), 8));
+    optionsContainer = createComposite(container, filling(new RowLayout(SWT.VERTICAL), true, true));
     createOptions();
```

Comment:
Would suggest adding `optionsContainer.setLayoutData(new GridData(SWT.CENTER, SWT.TOP, true, false));` here in this PR, to avoid text jumping at resizing.

Reply:
Done.